x11-misc/dunst/dunst-9999.ebuild

```diff
@@ -12,21 +12,23 @@ HOMEPAGE="https://dunst-project.org/ https://github.com/dunst-project/dunst"
 
 LICENSE="BSD"
 SLOT="0"
-IUSE="wayland"
+IUSE="+completions +dunstify wayland +X +xdg"
 
 DEPEND="
 	dev-libs/glib:2
 	sys-apps/dbus
-	x11-libs/cairo[X,glib]
+	x11-libs/cairo[X?,glib]
 	x11-libs/gdk-pixbuf:2
-	x11-libs/libX11
-	x11-libs/libXext
-	x11-libs/libXScrnSaver
-	x11-libs/libXinerama
-	x11-libs/libXrandr
-	x11-libs/libnotify
-	x11-libs/pango[X]
-	x11-misc/xdg-utils
+	x11-libs/pango[X?]
+	xdg? ( x11-misc/xdg-utils )
+	X? (
+		x11-libs/libX11
+		x11-libs/libXext
+		x11-libs/libXScrnSaver
+		x11-libs/libXinerama
+		x11-libs/libXrandr
+	)
+	dunstify? ( x11-libs/libnotify )
```

Comment:
Please look again through, this is also not sorted.

Reply:
ok done it.
is there an ebuild style guide so I don't have to make such mistakes?